scripts/test-lint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# ----------------------------------------------------------------------------------------------------------------------
+# This script is intended to contain all actions pertaining to code style checking, linting and normalisation.
+#
+# 1. The script executes linting routines on specific folders.
+# ----------------------------------------------------------------------------------------------------------------------
+
+# Stop on first error
+set -e;
+
+# banner
+echo -e "\033[93mLinting and style-checking...\033[0m";
+echo -en "\033[0m\033[2m";
+echo -e "eslint `eslint -v`\033[0m\n";
+
+# run style checker
+eslint index.js ./src/** ./test/**.js ;
```

Comment:
although we won't be having any non-js file inside the test folder, we should change it to `./test/**`